README.MD

```diff
@@ -7,106 +7,118 @@
+levels. Currently supported groups:
+    - Admins - users which have the AdministratorAccess policy attached, will be added to the admins group which will 
+    have the policy `arn:aws:iam::aws:policy/AdministratorAccess` attached
+    - PowerUsers - users which have write access to **any** of the services they use will be added to the powerusers 
+    group(s). In case of more than 10 policies being attached to that group, a number of groups will be created for 
+    powerusers, and the relevant users will be members of all of them.
+    - ReadOnly - users which only have read access to the account. Will be members of the readonly group which will have 
+    the policy `arn:aws:iam::aws:policy/ReadOnlyAccess` attached 
+- `terraform` - Creates Terraform files based on the configurations and the transformations applied by the optional
+arguments supplied
 
 ## Getting started
 
 ### Installation
 
 ```sh
-git clone https://github.com/bridgecrewio/AirIAM.git
-cd AirIAM/
-pipenv install --skip-lock
-pipenv shell
+pip install airiam 
 ```
 
-### Configuration
+### Run Configurations
 
-```sh
-python airiam.py -p <PROFILE> -u <NUM_OF_DAYS> -f <PATH_TO_DIR> -r
-```
-
-`profile / p` will use your custom profile and perform the initial checks over all regions when needed.
+#### General configurations for all commands
+- `profile / p` defines which profile to use for AWS access. If none specified, the AWS credentials will be taken from 
+the default places (environment variables, default profile etc.)
+- `last-unused-threshold / l` defines the threshold, in days, for entities to be marked as unused. Default value is 90
+- `no-cache` defines whether to reuse local data or re-scan. If this is the first time or the account was changed from
+the last run, cache invalidation will occur anyway.
 
-`unused / u` will define how many days will be considered as a least-usage threshold. By default value is 90, meaning, that any resource that was not active in the last 90 will be identified as unused and removed from the Terraform plan.
+#### `find_unused`
+- `output / o` defines the output format of the command. Default value is CLI.
 
-`folder / f` will define where the execution recovery plan and state files will be stored. By default, the files will be placed in a `results` folder.
+#### `recommend_groups`
+- `output / o` defines the output format of the command. Default value is CLI.
 
-`rightsize / r` will define whether to rightsize the permissions automatically using usage data from AWS or not.
+#### `terraform`
+- `directory / d` defines the output directory for the terraform code. Default value is `results/`
+- `ignore / i` defines a path to a text file listing regexp of entities which will be ignored by the terraform code 
+generator. 
+- `--without-unused` defines unused should not be moved to terraform.
 
-### Report
+### Running the commands
 
 ```sh
-***todo***
-```
+airiam find_unused -p dev -l 30 --no-cache
```

Comment:
using full command names  will make it clearer in this stage. shortcuts are easily selfed learned.